plugins/filter_aws/aws.h

```diff
@@ -106,11 +107,17 @@ struct flb_filter_aws {
     size_t account_id_len;
     int account_id_include;
 
-
     flb_sds_t hostname;
     size_t hostname_len;
     int hostname_include;
 
+    flb_sds_t *tag_keys;
+    size_t *tag_keys_len;
```

Comment:
why is this a pointer to a size_t, couldn't it just be a size_t

Reply:
Its an array of size_t since there are many tags. This confused me at first too.